style.css

```diff
@@ -0,0 +1,76 @@
+@import url("https://fonts.googleapis.com/css2?family=Josefin+Sans:wght@400;700&display=swap");
+
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  background-color: rgb(244, 244, 244);
+  font-family: "Josefin Sans", sans-serif;
+
+}
```

Comment:
One more thing, in your styling; please before starting to work on the project, configure `**Linters**` in your local environment. This will help you to discover and to correct small mistakes before committing and pushing the file into GitHub.  Hey 💯, No worries, if you forget how to use Linters into Html and Css, please be courageous to have a look at this link https://github.com/microverseinc/curriculum-transversal-skills/blob/main/clean-code/linters.md